packages/calcite-components/src/components/button/button.scss

```diff
@@ -339,6 +335,12 @@
     --calcite-internal-button-background-color: var(--calcite-color-foreground-1);
     @apply border-solid;
     border-width: var(--calcite-button-border-size, 1px);
+    &:hover {
+      @apply bg-foreground-2;
```

Comment:
Can you use our custom CSS props here (maps to `--calcite-color-foreground-2` and `--calcite-color-foreground-3` respectively)? We're moving away from Tailwind utils, so new code can use CSS vars directly.